internal/host/validations_test.go

```diff
@@ -289,5 +304,176 @@ var _ = Describe("Validations test", func() {
 			Expect(validationStatus).To(Equal(ValidationSuccess))
 			Expect(validationMessage).To(Equal(fmt.Sprintf("Installation disk can be encrypted using %s", models.DiskEncryptionModeTpmv2)))
 		})
+
+		It("day2 host - TPM2 for worker enabled on cluster, TPM2 available on host", func() {
+			c := hostutil.GenerateTestCluster(clusterID, common.TestIPv4Networking.MachineNetworks)
```

Comment:
you can put some of the tests under a single context and have the common code written once in before each

Reply:
yeah, didn't want to make too much refactoring as part of this change. I'll try to extract some common code to funcs.